.github/workflows/publish.yml

```diff
@@ -51,6 +52,8 @@ jobs:
     # Scan image for vulnerabilities
     - name: Aqua Security Trivy image scan
       run: |
+        # Ignore Vulnerabilities
+        printf ${{ env.IGNORE_VULNS }} > .trivyignore
```

Comment:
Use repo vars for whitelist?

Reply:
yeah guess i need perms on this

Reply:
i suppose the other thing is when we need to update these repo vars for the allowlist, it doesn't keep a history of what's in there so maybe a good feature anyways just to re-evaluate when we need to add or remove them

Reply:
donneeeeee